src/assets/style/airport.css

```diff
@@ -65,22 +65,30 @@
   width: 5em;
   font-weight: bold;
   font-size: 120%;
-  margin-right: 0.25em;
   text-align: right;
   white-space: nowrap;
 }
 
 #airport-switch li .icao {
   display: inline-block;
-  width: 4em;
-  /*margin-right: */
+  width: 2.5em;
+  margin-left: 1em;
+  text-align: center;
+  font-weight: bold;
+}
+
+#airport-switch li .symbol {
+  display: inline-block;
```

Comment:
Property order is a thing with CSS rules, but these are all going to be refactored with #481, so you can leve this as-is for now.

https://github.com/thenerdery/html-css-standards/blob/master/standards/css.md#property-order